apitest/pom.xml

```diff
@@ -20,11 +20,18 @@
 	<scm>
 		<connection>scm:git:git://github.com/mosip/mosip-functional-tests.git</connection>
 		<developerConnection>scm:git:ssh://github.com:mosip/mosip-functional-tests.git</developerConnection>
```

Comment:
please update the repo url here

Reply:
Updated the URL hence resolving Conversation.